packages/uploadthing/src/internal/handler.ts

```diff
@@ -357,13 +358,16 @@ const handleUploadAction = Effect.gen(function* () {
     generateUploadThingURL("/v6/prepareUpload"),
     {
       method: "POST",
-      body: JSON.stringify({
-        files: filesWithCustomIds,
-        routeConfig: parsedConfig,
-        metadata,
-        callbackUrl: callbackUrl.origin + callbackUrl.pathname,
-        callbackSlug: opts.slug,
-      }),
+      body: JSON.stringify(
```

Comment:
is this the only place we need to use this replacer? how bout when sending route config to client?

Reply:
Ah good catch